tests/drivers/can/api/src/canfd.c

```diff
@@ -395,6 +434,31 @@ ZTEST_USER(canfd, test_set_timing_data_min)
 	zassert_equal(err, 0, "failed to start CAN controller (err %d)", err);
 }
 
+/**
+ * @brief Test setting a too low data phase bitrate.
+ */
+ZTEST_USER(canfd, test_set_bitrate_data_too_low)
```

Comment:
There is already negative test that checks error code returned by can_set_bitrate_data() when bitrate value is too high:
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/can/api/src/canfd.c#L401
Here, I add similar test. It checks that can_set_bitrate_data() returns -ENOTSUP when bitrate value is too low.